File: Add Code Here/C++/14. Longest Common Prefix.cpp
    string longestCommonPrefix(vector<string>& strs) {
        
        sort(strs.begin(), strs.end());
       
        int i=0, j=0, n=strs.size();
        string ans="", a=strs[0], b=strs[n-1];
       
        while(i < a.size() && j < b.size()){
            if(a[i]!=b[j]) break;
            else {
                ans+=a[i];
                i++,j++;
            }
        }
    
        return ans;
    }

File: Add Code Here/JAVA/LeetCode/14. Longest Common Prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String temp=strs[0];
        for(int i=0; i<strs.length; ++i){
            while(strs[i].indexOf(temp)!=0){
                temp=temp.substring(0,temp.length()-1);
                if(temp.isEmpty()) return "";
            }
        }
        return temp;
    }
}
